src/components/Products.js

```diff
@@ -0,0 +1,16 @@
+import ProductCard from "./ProductCard"
+import React from 'react'
+
+const Product = ({students}) => {
+    return (
+        <section>
+            {students.map((student, i) => (
+                <ProductCard {...student} key={student.id} />
```

Comment:
Looks like you have good intentions here of rendering a list of `<Product/>` components. See my other comment above in `App.js` - you aren't rendering `Products`